charts/pega/README.md

```diff
@@ -1165,7 +1165,7 @@ Parameter   | Description   | Default value
 `service.tls.traefik.insecureSkipVerify` | Set to `true` to skip verifying the certificate; do this in cases where you do not need a valid root/CA certificate but want to encrypt load balancer traffic. Leave the setting to `false` to both verify the certificate and encrypt load balancer traffic. | `false`
 
 ##### Important Points to note
-- By default, Pega provides a self-signed keystore and a custom root/CA certificate in Helm chart version `2.2.0`. To use the default keystore and CA certificate, leave the parameters service.tls.keystore, service.tls.keystorepassword and service.tls.cacertificate empty.
+- By default, Pega provides a self-signed keystore and a custom root/CA certificate in Helm chart version `2.2.0`. To use the default keystore and CA certificate, leave the parameters service.tls.keystore, service.tls.keystorepassword and service.tls.cacertificate empty.Expiry date of default keystore and CA certificate is 25/12/2025.
```

Comment:
can you make some slight improvements for readabilty?
service.tls.cacertificate empty.Expiry date
to
service.tls.cacertificate empty.*** The expiration*** date

Reply:
@taz-pega-work They are different statements . We are shipping default keystore and CA certificate which has some expiry date which I mentioned.